plugins/interactivity-api-countdown-3cd73e/src/view.js

```diff
@@ -1,34 +1,36 @@
 import { store, getContext } from '@wordpress/interactivity';
 
 store( 'interactivity-api-countdown-3cd73e__store', {
+	state: {
+		get seconds() {
+			const { remaining } = getContext();
+			return remaining % 60;
+		},
+		get minutes() {
+			const { remaining } = getContext();
+			return Math.floor( remaining / 60 ) % 60;
+		},
+		get hours() {
+			const { remaining } = getContext();
+			return Math.floor( remaining / 3600 ) % 24;
+		},
+		get days() {
+			const { remaining } = getContext();
+			return Math.floor( remaining / 86400 );
+		},
```

Comment:
Now that these properties are getters inside `state`, the references used by directives need to be updated accordingly. So, we'd need to replace `context.` with `state.` in the following lines:

https://github.com/WordPress/block-development-examples/blob/8f559dd3c654694742b2fea684ca6bad69992942/plugins/interactivity-api-countdown-3cd73e/src/render.php#L36-L39